internal/fs/inode/dir.go

```diff
@@ -210,7 +210,7 @@ func NewDirInode(
 	bucket *gcsx.SyncerBucket,
 	mtimeClock timeutil.Clock,
 	cacheClock timeutil.Clock,
-	typeCacheMaxEntries int) (d DirInode) {
+	typeCacheSizeInMbPerDirectory int) (d DirInode) {
```

Comment:
we can remove "PerDirectory" from the name.

Reply:
removed